contributions/api/utils/query_params.py

```diff
@@ -17,7 +17,7 @@
 def format_query_contribution(args, query):
     query_parts = []
 
-    if args.get('id'):
+    if args.get('eventId'):
```

Comment:
Please remove references to eventId.

Reply:
> Please also update the functions that implement these endpoints. The query parameter can be received in the function directly using parameter.
> For e.g., `def allcapabilitiessearch(name=None):`

I am working on this but if we do this, the code will be much more complicated. Is there any benefit of doing this?

Reply:
for example, if there the name is none, I am trying to return all records assuming that as no argument. However, this doesn't catch the bad argument, like the argument is 'name123'. This will return the list of every record instead of reporting as bad argument. To remove this, I have to put some handling method of the argument is the 'name' or if there is no argument... That will be almost similar to the currently code that uses argument but more complex and harder to understand.

Reply:
I pushed the changes with (name=None)

Reply:
> Please remove references to eventId.

removed